src/concrete/ml/torch/hybrid_model.py

```diff
@@ -511,7 +516,15 @@ def forward(self, x: torch.Tensor, fhe: str = "disable") -> torch.Tensor:
         for module in self.remote_modules.values():
             module.executor = self.executor
 
-        result = self.model(x)
+        # Initialize the progress bar based on the number of remote modules
+        num_remote_modules = len(self.remote_modules)
+        with tqdm(total=num_remote_modules, desc="Remote Modules Inference") as pbar:
```

Comment:
compilation also has a tqdm on it, could your callback be used there too ?

`        for name in tqdm(self.module_names, desc="Compiling FHE layers"):`